<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RelacaoColaborador</class>
 <widget class="QWidget" name="RelacaoColaborador">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="painelTabela">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="campoPesquisarObjetosTabela"/>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QGroupBox" name="painelResultados">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="campoItemSelecionado">
        <property name="text">
         <string>[]</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="campoTotalRegistros">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QFrame" name="frameFiltros">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QLineEdit" name="campoID_Empresa">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/images/flag_otimo.png&quot;/&gt;Código da &lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;empresa&lt;/span&gt; que deseja filtrar. Nesta versão só uma empresa é permitida.&lt;/p&gt;&lt;p&gt;Caso queira todas as empresas, deixe o campo em branco/vazio.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>Código da Empresa</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="campoID_Filial">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/images/flag_otimo.png&quot;/&gt;Código da &lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;filial&lt;/span&gt; que deseja filtrar. Nesta versão só uma empresa é permitida.&lt;/p&gt;&lt;p&gt;Caso queira todas as empresas, deixe o campo em branco/vazio.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>Código da Filial</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frameAcoes">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QDateEdit" name="dataReferencia">
        <property name="wrapping">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="showGroupSeparator" stdset="0">
         <bool>true</bool>
        </property>
        <property name="currentSection">
         <enum>QDateTimeEdit::DaySection</enum>
        </property>
        <property name="displayFormat">
         <string>dd/MM/yyyy</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="botaoProcessar">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>Processar</string>
        </property>
        <property name="icon">
         <iconset resource="thema.qrc">
          <normaloff>:/images/paper-plane.png</normaloff>:/images/paper-plane.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="botaoExportar">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>Exportar</string>
        </property>
        <property name="icon">
         <iconset resource="thema.qrc">
          <normaloff>:/images/csv-file-format-extension.png</normaloff>:/images/csv-file-format-extension.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>campoID_Empresa</tabstop>
  <tabstop>campoID_Filial</tabstop>
  <tabstop>dataReferencia</tabstop>
  <tabstop>botaoProcessar</tabstop>
  <tabstop>botaoExportar</tabstop>
  <tabstop>campoPesquisarObjetosTabela</tabstop>
  <tabstop>tableWidget</tabstop>
 </tabstops>
 <resources>
  <include location="thema.qrc"/>
 </resources>
 <connections/>
</ui>
